Dependencies/UrlLib/Source/Android/UrlRequest.cpp

```diff
@@ -45,7 +45,8 @@ namespace UrlLib
         void Open(UrlMethod method, std::string url)
         {
             m_method = method;
-            Uri uri{Uri::Parse(std::move(url).data())};
+            m_url = std::move(url);
+            Uri uri{Uri::Parse(m_url.data())};
             m_schemeIsApp = ((std::string)uri.getScheme() == "app");
             m_urlOrAppPath = uri.getPath();
```

Comment:
```suggestion
            if ((std::string)uri.getScheme() == "app")
            {
                m_schemeIsApp = true;
                m_urlOrAppPath = uri.getPath();
            }
            else
            {
                m_urlOrAppPath = std::move(url);
            }
```

and revert the other stuff